python/cudf/cudf/io/json.py

```diff
@@ -198,6 +193,7 @@ def read_json(
                 mixed_types_as_string=mixed_types_as_string,
                 prune_columns=prune_columns,
                 recovery_mode=c_on_bad_lines,
+                extra_parameters=kwargs,
```

Comment:
Just an FYI. I'm updating this API to use json reader options classes to match the other IO functions (eg. https://github.com/rapidsai/cudf/blob/branch-25.02/python/pylibcudf/pylibcudf/io/parquet.pyx#L309). So it will look like
```python
plc.io.json.read_json(
    plc.io.json.JsonReaderOptions.builder(
        plc.io.SourceInfo(file_paths_or_buffers)
    )
    .byte_range_size(...)
    ...
)
```

Reply:
These options will change because most of options are for spark, and they will change until all spark json feature requests are completed. These options are not intended for python user mostly. It's exposed for quicker testing.